Added PIP Install Commands for Python Modules

tso914 opened the pull request:
Added PIP Install commands for the Python Modules we use.

tso914 committed:
Added PIP Install Commands for Python Modules

Added PIP Install commands for the Python Modules we use.

tso914 commented:
New PIP Install Commands for Python modules

stephenconnolly1 commented on Dockerfile:
```diff
@@ -1,48 +1,60 @@
-
-# bash as a default command (but allow to be overridden by anything else supplied on CMD line)
-CMD bash
+FROM centos:latest
```
@tso914  - not sure what's gone on here, but the diffs are showing up as Deletes and Adds. Can you check that you haven't changed the line-ending settings on the dockerfile.

stephenconnolly1 commented on Dockerfile:
```diff
@@ -1,48 +1,60 @@
+    make install
+
+# Install pypostal
+RUN pip3.6 install postal
```
Each RUN command creates a new image layer, which can be expensive and slow.
Might be more efficient to have 


    RUN pip3.6 install postal \
        happybase \
        pandas \ 
      ...

tso914 committed:
Corrected CR/LF and Single Run Command

Corrected CR/LF to LF and all PIP installs now in single run command

tso914 committed:
Corrections as per Stephens suggestion

Corrected CR/LF to LF and also add all the PIP install commands in a single run command.

tso914 commented:
Hi Stephen, 

I have made the changes to the end of lines and also corrected the PIP Install command to be in one run. I acidently uploaded a .txt version of the file which I don't seem to be able to delete. Is it a permission thing? can you delete it?

Thanks,

Commit:
removing dockerfile.txt. Not needed

stephenconnolly1 reviewed (approved):
Changes made as required

stephenconnolly1 commented:
@tso914 - Changes approved and merged. The build should appear in Docker Hub in about an hour or so. I would suggest you pull that down on your laptop, save as a tarball, import into the env and then within Dev, do a Docker load and then docker push up to Artifactory. 
Give me a shout if you need help - the docker commands can be a bit fiddly, especially for pushing up to artifactory with the right label.
